src/main/ank/1.7-functors-Part1.md

```diff
@@ -63,7 +68,10 @@ Nothing
 id Nothing
 ```
 ```markdown
-fmap {identity(it)} (None)
+import arrow.core.None
+import arrow.core.identity
+
+fmap(f = ::identity, o = None)
```

Comment:
```suggestion
fmap { identity(it) } (None)
```

Reply:
🤔The method signature of `fmap` has changed in the doc
`fun <A, B> fmap(f: (A) -> B, o: Option<A>): Option<B>`
Won't that make `fmap { identity(it) } (None)` invalid?

Reply:
which doc are you always referring to? 
The aim of this project is to translate the book.

Reply:
If you referring to the Arrow-docs. The nearest what we try to model is this [function](https://github.com/arrow-kt/arrow/blob/ffd5409f6ce0594d61149d4c68964f27b651bb2a/modules/core/arrow-core-data/src/main/kotlin/arrow/core/Option.kt#L81) here.